VIPER-SWIFT/Classes/Common/Categories/NSCalendar+CalendarAdditions.swift

```diff
@@ -8,129 +8,130 @@
 
 import Foundation
 
-extension NSCalendar {
-    class func gregorianCalendar() -> NSCalendar {
-        return NSCalendar(calendarIdentifier: NSCalendarIdentifierGregorian)!
+extension Calendar {
+    static func gregorianCalendar() -> Calendar {
```

Comment:
I realize I didn't follow new naming conventions at all ! "gregorian" is definitely a better name